client/src/app/landing/sections/Components/PriceComponent.tsx

```diff
@@ -122,56 +122,44 @@ type PriceSectionModelSelectorProps = {
 function PriceSectionModelSelector(props: PriceSectionModelSelectorProps) {
   const { models, selectedModelIndex, setSelectedModelIndex } = props;
 
+  const numberOfModels = models?.length ?? 0;
+
   return (
-    <div className='flex flex-row items-center justify-center overflow-x-auto w-full scrollbar-hide'>
+    <div className='flex flex-row items-center sm:justify-center justify-start overflow-x-auto w-full scrollbar-hide border-b border-gray-200'>
       {models?.map((model, index) => (
         <PriceSectionModelSelectorItem
           key={model.id}
           model={model}
           isSelected={selectedModelIndex === index}
           onClick={() => setSelectedModelIndex(index)}
+          numberOfModels={numberOfModels}
         />
       ))}
     </div>
   );
 }
 
-type PriceSectionProps = {
+type Props = {
   className?: string;
 };
 
-export function PriceSection(props: PriceSectionProps) {
+export function PriceComponent(props: Props) {
   const { className } = props;
 
   const { models } = useOrFetchModels();
-  const threeFirstModels = models?.slice(0, 3);
+  const threeFirstModels = models?.slice(0, 4);
```

Comment:
careful, the reference might change on each render.
You can likely remove entirely for https://linear.app/workflowai/issue/WOR-4148/additional-models-in-price-match-section-on-landing-page